blueprints/app/files/package.json

```diff
@@ -64,7 +64,7 @@
     "webpack": "^5.72.0"
   },
   "engines": {
-    "node": "12.* || 14.* || >= 16"
+    "node": "14.* || >= 16"
```

Comment:
@bertdeblock should this actually become `14.* || 16.* || >= 18`? Node.js 17 is EOL as well as it's non LTS

Reply:
actually, worth having it in separate PR as we should add v18 elsewhere, like `docs/node-support.md`, `.github/workflows/ci.yml`, etc.

Reply:
Hi @SergeAstapov, it was indeed my intention to add support for v18 in a separate PR.
There's also a separate ticket for it: https://github.com/ember-cli/ember-cli/issues/9905.